test/WebSites/ValidationWebSite/gruntfile.js

```diff
@@ -0,0 +1,21 @@
+
+module.exports = function (grunt) {
+    grunt.initConfig({
```

Comment:
When you get this in, could you send an email to the team on how to install npm, grunt and bower globally? It would avoid having to download these packages after every clean.